TvMaterialCatalog/app/src/main/java/com/google/tv/material/catalog/Component.kt

```diff
@@ -1,34 +1,34 @@
 package com.google.tv.material.catalog
 
+val foundations = listOf(
+    Component.Color,
+    Component.Typography,
+    Component.Motion,
+    Component.Interaction,
+)
+
 val components = listOf(
     Component.Button,
     Component.Card,
     Component.Chip,
     Component.ListItem,
     Component.ImmersiveList,
     Component.FeaturedCarousel,
-    Component.NavigationDrawer,
     Component.TabRow,
+)
+
+val componentsPlanned = listOf(
+    Component.NavigationDrawer,
```

Comment:
NavigationDrawer is part of Beta right?

Reply:
Yes, but this is just a placeholder and the implementation is not there yet.